specifyweb/frontend/js_src/lib/localization/resources.ts

```diff
@@ -733,28 +733,13 @@ export const resourcesText = createDictionary({
     'uk-ua': 'Запустити в колекції',
   },
   createNewForm: {
-    'en-us': 'Create new form',
-    'de-ch': 'Definition',
-    'es-es': 'Crear nuevo formulario',
-    'fr-fr': 'Créer un nouveau formulaire',
-    'ru-ru': 'Создать новую форму',
-    'uk-ua': 'Створити нову форму',
+    'en-us': 'Create a new view definition',
```

Comment:
Hmm, I was actually hoping to retire `view definition` in favor of `form`

I.e `Form Editor` is clear to everyone
`View definition editor` is long, a pain to translate, and not clear
plus, it's ambiguous - `view the editor of definitions` or `the editor for view definitions`

view is a term that comes from sp6 and java. forms comes from html

Reply:
https://github.com/specify/specify7/pull/4867#issuecomment-2101313908